pom.xml

```diff
@@ -133,7 +133,7 @@
     <hive.parquet.version>1.6.0</hive.parquet.version>
     <jetty.version>9.3.20.v20170531</jetty.version>
     <javaxservlet.version>3.1.0</javaxservlet.version>
-    <chill.version>0.8.4</chill.version>
+    <chill.version>0.9.2</chill.version>
```

Comment:
I am not sure whether this is should be changed. If it is unreasonable, I can change it back.

Reply:
Why do you need to update it?

Reply:
Not necessary. Chill 0.9.2 uses kryo 4.0. I can change it back.

Reply:
please change it back.

Reply:
library upgrading deserves a separated PR.